torch/csrc/jit/init.cpp

```diff
@@ -185,6 +185,16 @@ void initJITBindings(PyObject* module) {
                   param_name,
                   getQParamFunc,
                   at::ScalarType::QInt8);
+            } else if (param_name == std::string("bias")) {
```

Comment:
The name `bias` is hardcoded here (as `weight` earlier), which makes this pass only applicable to `Conv` ops. We should either rename the pass to `insert_quantdequant_for_conv` or find a way to avoid such hardcoding. Otherwise, the name is a lie.

Reply:
Even FC also

Reply:
yes planning to rename it explicitly as _jit_pass_insert_quantdequant_for_weight_bias